### ISSUE TITLE
Renegade Ops [NPUB30406]: Freezes on boot

### ISSUE THREAD

**Issue by Shoegzer** [RPCS3/rpcs3#14899]
### Quick summary

This title freezes on boot due to an issue processing the license (rap/edat) files.

### Details

The following is published to the log:
```
E sceNp: npDrmIsAvailable(): Rap file not found: “/home/testing/.config/rpcs3/dev_hdd0/home/00000001/exdata/UP0177-NPUB30406_00-RENEGADEOPSKEY00.rap”
E SYS: 'sceNpDrmIsAvailable' failed with 0x80029521 : SCE_NP_DRM_ERROR_LICENSE_NOT_FOUND, /dev_hdd0/game/NPUB30406/USRDIR//KEY.EDAT [1]
```
Note the following:
* Both files have been confirmed to exist in the locations specified in the above log excerpt
* The double-slash before KEY.EDAT in the above log excerpt should be a single slash?
* Removing  `/dev_hdd0/game/NPUB30406/USRDIR/KEY.EDAT` will cause the game to boot, however of course only as a demo.

### Attach a log file

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/13690267/RPCS3.log)

### System configuration

* OS: Linux Mint 21.2 / kernel 6.15
* CPU: Intel i7-10870H
* GPU: Nvidia GTX 3070 / driver 535.129
* Commit: 47fcb9562fd531350f7d9c847dda322c5f50d56d

**Comment by Shoegzer** [comment]
Update: Tested against PR #14908.  I assume this doesn't require re-installation of the rap/edat files so I left them installed from previous tests.  The rap/edat specific errors no longer appear in the log, though the title still fails to boot.  

Log file: [RPCS3.log](https://github.com/RPCS3/rpcs3/files/13710619/RPCS3.log)

**Comment by Megamouse** [comment]
I didn't really change anything. So you must be doing something different.
Try removing the game data and the rap and try the PR from scratch.

**Comment by Shoegzer** [comment]
Thanks.  I tried from scratch but it's still freezing on boot. Here is a log showing the entire installation process, which includes installing the game, its "unlock key" pkg, and the unlock key's corresponding rap file, and then booting the game to freezing:

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/13711052/RPCS3.log)

AFAICT the main problem here is that rpcs3 installs the unlock key pkg (KEY.EDAT) to `/dev_hdd0/game/NPUB30406/USRDIR/` for some reason, rather than `/dev_hdd0/home/00000001/exdata` as it does with other titles. At least, by removing KEY.EDAT the title will boot, though of course only in demo mode because it expects an encrypted license file.  I tried manually moving the file to `/dev_hdd0/home/00000001/exdata` but the title is still seen as a demo.

**Comment by Megamouse** [comment]
That makes no sense. The EDAT is part of game data and not exdata.
Anyway, your log shows no errors, which means we still don't see the original behavior.

**Comment by Megamouse** [comment]
The log shows that either your game or your license are corrupted

**Comment by Shoegzer** [comment]
That's true, titles should install edats to the game data folder because licenses are technically part of a game's data.  That said, I've seen some install them to exdata.  I have no idea why, it doesn't make sense to me either thinking about it now, but there they are.

Also where in the log are you seeing that the game or license is corrupted?  At least the checksums of both match those of known good dumps.  Perhaps rpcs3 is somehow corrupting the data on install?

**Comment by Shoegzer** [comment]
I looked through the log trying to find indications of a corrupt file, the closest I could find was what appears to be a failure to load assets, such as below.  Is this what you meant?

`·E 0:02:12.903682 {PPU[0x1000000] Thread (main_thread) [0x001cf70c]} sys_fs: 'sys_fs_open' failed with 0x80010006 : CELL_ENOENT, “/dev_hdd0/game/NPUB30406/USRDIR/archives_ps3/game3.tab” [1]`

Again though, the title runs fine without the edat file in USRDIR (even though those same failures appear in the log).  For comparison, here's a log from successful launch:

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/13734139/RPCS3.log)

**Comment by Shoegzer** [comment]
Issue persists as of commit 56d35bf409d54bc82cdf92aa7f1be88b27899d2a.  Game and license files have been confirmed as good.  The KEY.EDAT is installed to the game-specific USRDIR directory upon drag-and-drop of the unlock .pkg file to RPCS3.  AFAICT this is the only file the .pkg installs.  I've looked closer through the logs but haven't found any evidence of corruption other than possibly what I posted above, though I wonder if the issue could be due to case-sensitivity with this Linux test system?

The game apparently works under Windows at least, as can be seen [here](https://www.youtube.com/watch?v=69K6oH0PYUQ).

**Comment by Shoegzer** [comment]
This title installs fine now under the latest with a clean reinstall.  There are ingame crashes, however out of scope for this issue.
